README.md

```diff
@@ -1,3 +1,5 @@
+if you want to use KNN-CAD algorithm, please make reference to our [article](https://arxiv.org/abs/1608.04585).
```

Comment:
Please remove this line. I will make a note to this effect in the README after merging this PR.